client/src/components/molecules/style/RequestsTable.scss

```diff
@@ -0,0 +1,42 @@
+.request-table {
+    @include table-base;
+    h1 {
+        @include section-header-text;
+        padding-bottom: 4px;
+        margin: 0;
+    }
+    width:1200px;
+    margin-left: auto;
+    margin-right:auto;
+    .edit{
+        color: #333333;
```

Comment:
Use colors from _config.scss whenever possible.
But also, @mixin table-base already sets the color for all "a" elements.

Reply:
For some reason it wasn't working.

Reply:
Merge with main and it will work. My branch with this was merged in a week or so ago